features/polyPinion/src/context/questionnaire-list-context.tsx

```diff
@@ -186,7 +192,7 @@ export const QuestionnaireListProvider: React.FC = ({ children }) => {
             value={{
                 triggerUpdate,
                 saveQuestionnaireAnswers,
-                markQuestionaireSubmitted,
+                markQuestionnaireSubmitted: markQuestionnaireSubmitted,
```

Comment:
Isn't this redundant and could just be `markQuestionnaireSubmitted`?

Reply:
👀 - corrected here and above 👍🏽 
thank youu